app/views/catalog/_metadata.html.erb

```diff
@@ -0,0 +1,12 @@
+<% document ||= @document %>
+
+<div class='metadata-view'>
+  <% document.references.refs.each do |ref| %>
+    <% if Settings.METADATA_SHOWN.include?(ref.type.to_s) %>
+      <div class='label label-default'>
+        <%= formatted_name_reference(ref.type) %>
+      </div>
+      <%= CodeRay.scan(Geoblacklight::Metadata.new(ref).metadata, :xml).div(css: :class).html_safe %>
```

Comment:
What do you think about adding a tab_width option:

``<%= CodeRay.scan(Geoblacklight::Metadata.new(ref).metadata, :xml).div(css: :class, tab_width: 2).html_safe %>``

The `columbia-columbia-landinfo-global-aet` layer, for example, has 8 spaces for tabs which puts most of the XML off the visible pane requiring a lot more horizontal scrolling.